src/intl/en/page-upgrades-get-involved-bug-bounty.json

```diff
@@ -60,21 +66,21 @@
   "page-upgrades-bug-bounty-type-3": "Inconsistencies in assumptions, like situations where honest validators can be slashed",
   "page-upgrades-bug-bounty-type-4": "Calculation or parameter inconsistencies",
   "page-upgrades-bug-bounty-types": "Types of bugs",
-  "page-upgrades-bug-bounty-validity": "Valid bugs",
-  "page-upgrades-bug-bounty-validity-desc": "This bug bounty program is focused on finding bugs in the core consensus layer Beacon Chain specification and the Lighthouse, Nimbus, Teku, Prysm, and Lodestar client implementations.",
+  "page-upgrades-bug-bounty-validity": "In Scope",
+  "page-upgrades-bug-bounty-validity-desc": "Our bug bounty program spans end-to-end: from soundness of protocols (such as the blockchain consensus model, the wire and p2p protocols, proof of work, proof of stake, etc) and protocol/implementation compliance to network security and consensus integrity. Classical client security as well as security of cryptographic primitives are also part of the program. When in doubt, send an email to bounty@ethereum.org and ask us.",
   "page-upgrades-bug-bounty-card-critical": "Critical",
   "page-upgrades-bug-bounty-card-critical-risk": "Submit critical risk bug",
   "page-upgrades-bug-bounty-card-h2": "Medium",
   "page-upgrades-bug-bounty-card-high": "High",
   "page-upgrades-bug-bounty-card-high-risk": "Submit high risk bug",
   "page-upgrades-bug-bounty-card-label-1": "Up to 1,000 points",
-  "page-upgrades-bug-bounty-card-label-2": "Up to 2,000 DAI",
+  "page-upgrades-bug-bounty-card-label-2": "Up to 2,000 USD",
```

Comment:
We should leave the max points the same. This will ensure the leaderboard does not get hugely skewed. It's that we are changing the points to $ conversion.

So 25k is max points but you now gt $10 per point